### DISCUSSION TITLE
How should my rector.php look like in a php 7.3 project?

### DISCUSSION THREAD

**Discussion by ziming** [rectorphp/rector#7367] [Q&A]
Currently if we look at the generated rector.php template, it seem to use language features above php 7.3

https://github.com/rectorphp/rector/blob/main/templates/rector.php.dist

Other than removing the void keyword, how should i modify it so that the rector.php file is PHP 7.3 valid? and still work?

**Comment by TomasVotruba** [comment]
> it uses language features above php 7.3

What exact features do you mean? The `void` is available since PHP 7.1. https://www.php.net/manual/en/migration71.new-features.php

What error do you get when you run Rector with this config?

**Reply by ziming** [reply]
my bad, i saw the github action error message wrongly.

but just curious, this is valid in php 7.3?

return static function ...

**Reply by TomasVotruba** [reply]
What is the result you get from your PHP version when you run it?

**Reply by samsonasik** [reply]
static function exists since PHP 5.4 https://3v4l.org/MAeXLi#v5.4.45

**Reply by ziming** [reply]
thanks for the site. as I said earlier I see wrongly from github action.

nice to know I can just write return outside of any class or function in a file.
